src/components/NavLinks/SideBarLinks.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { makeStyles, Link, List, ListItem, ListItemIcon, ListItemText, createMuiTheme, ThemeProvider } from "@material-ui/core";
+import { styles } from "../../styles/styles";
+import IconMaker from '../../sections/SideBar/components/IconMaker/IconMaker';
```

Comment:
If this is a component for all the aplications there shouldn't be anything imported from sections, in any case this is another component or a subcomponent of SideBarLinks